### PR TITLE
Cleanup and docs

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Comment by samdark** [comment]
Questions/TODO:

1. There are `.env` and `.env.test`. Seems these aren't used in any way. What was the purpose? @terabytesoftw 
2. <strike>Should we remove Scrutinizer?</strike>
3. Currently, we enable/disable debug via environments, but there's YII_DEBUG at the same time in `index.php`. What was the idea? @terabytesoftw 
4. Should we link to https://github.com/yiisoft/yii-runner-console and review `yii` entry script?
5. Describe events at https://github.com/yiisoft/yii-console and exit codes.

**Review by terabytesoftw — APPROVED** [review]

File: .env
Comment (vjik) [inline .env:1]: Should `.env` to be in git ignore?

File: README.md [hunk 1]
```diff
@@ -9,12 +9,18 @@
 [![Latest Stable Version](https://poser.pugx.org/yiisoft/app-console/v/stable.png)](https://packagist.org/packages/yiisoft/app-console)
 [![Total Downloads](https://poser.pugx.org/yiisoft/app-console/downloads.png)](https://packagist.org/packages/yiisoft/app-console)
 [![Build status](https://github.com/yiisoft/app-console/workflows/build/badge.svg)](https://github.com/yiisoft/app-console/actions?query=workflow%3Abuild)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/app-console/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/app-console/?branch=master)
-[![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/app-console/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/app-console/?branch=master)
```
Comment (vjik) [inline README.md:13]: Add codecov badge with code codevarge

File: README.md [hunk 2]
```diff
@@ -25,33 +31,119 @@ The package is a console application that can be used to perform common tasks in
...
+$ ./yii echo
+You said: Hello!
+
+$ ./yii echo 'Code something'
+You said: Code something
+```
+
+## Environments
+
+Out of the box, three environments are available:
```
Comment (vjik) [inline README.md:102]: Need to remove yiisoft/yii-debug params from configration

**Review by vjik — CHANGES_REQUESTED** [review]

File: .env
Comment (samdark) [reply .env:1]: Not really. It's used for Codeception only and is immediately overwritten.

File: README.md (hunk 2 above)
Comment (samdark) [reply README.md:102]: Done.

File: README.md (hunk 1 above)
Comment (samdark) [reply README.md:13]: Can you help with it?

Comment (vjik) [reply README.md:13]: I missing that template use Codeception, but not PhpUnit.

**Review by vjik — APPROVED** [review]

**Pull request merged**